### PR TITLE
VBID-9565 Change naming scheme for file outputs

### PR DESCRIPTION
**Pull request by judebusarello**

### CODE REVIEW

**Comment by chrismcbride** [comment]
+1

**Comment by psimmons-vistar** [comment]
Whats the context for this change?  Do we need to worry about this breaking any other repos that depend on vistar-gulp-tasks?  Should we maybe make the output for css/js configurable instead?

**Comment by judebusarello** [comment]
The context is that shared maps are broken in part by the names of the files being changed for deploys recently. So instead of having a bunch of "if" statements to see if we're doing local dev, we're just going to consolidate the names to be the same again.

I think we only used gulp for local dev, but I'm not an authority.

**Comment by psimmons-vistar** [comment]
Ah, I also totally forgot that versioning will make it so that this won't impact any other of our existing repos which depend on vistar-gulp-tasks

**Comment by psimmons-vistar** [comment]
:thumbsup:

File: src/index.coffee [hunk 1]
```diff
@@ -109,7 +109,7 @@ module.exports = (projectConfig={}) ->
       .pipe(source(config.browserify.entries[0]))
       .pipe(buffer())
       .pipe(ugly())
-      .pipe(concat('app.js'))
+      .pipe(concat('development-bundle.js'))
```
Comment (markchadwick) [inline src/index.coffee:112]: Under which circumstances do we serve this? Only local development?

Comment (judebusarello) [reply src/index.coffee:112]: Yeah, we only use gulp for local development.

**Comment by cliscum** [comment]
+1

**Comment by judebusarello** [comment]
Anyone want to give me the ole' Plus Two on this guy?

**Comment by cliscum** [comment]
+2

**Pull request merged**